raiden/tests/smart_contracts/test_channel_manager.py

```diff
@@ -102,7 +102,8 @@ def test_channelmanager(
         channel_manager.newChannel(address1, settle_timeout)
 
     # should be false if there is no channel for the given address
-    assert not channel_manager.getChannelWith(inexisting_address)[1]
+    assert channel_manager.getChannelWith(inexisting_address)[0] == '0' * 40
```

Comment:
I know you did not introduce the term but can you please correct `inexistent` to `nonexistent`?